lib/Phpfastcache/Extensions/Drivers/Couchbasev4/Driver.php

```diff
@@ -40,15 +49,23 @@
 use Phpfastcache\Exceptions\PhpfastcacheUnsupportedMethodException;
 use ReflectionExtension;
 
+use function bin2hex;
+use function class_exists;
+use function random_bytes;
```

Comment:
Prefer using absolute calls to functions rather than importing them :)
C.f [Coding guidelines](https://github.com/PHPSocialNetwork/phpfastcache/blob/master/CODING_GUIDELINE.md#php-core-classes)